Schema/PublicSchema/PerformanceCounterData.bond

```diff
@@ -0,0 +1,33 @@
+import "Domain.bond"
+import "DataPointType.bond"
+
+namespace AI
+
+[Description("Instance of Performance Counter represents data for a specific Windows Performance Counter.")]
+struct PerformanceCounterData
+    : Domain
+{
+    [Name("version")]
+    10: required int32 	 ver = 2;
+    
+    20: required string 	 categoryName;
```

Comment:
those will be deprecated. I generated them so deprecation will be a separate PR that easier to review